src/java.desktop/share/classes/sun/awt/SunToolkit.java

```diff
@@ -1431,7 +1431,7 @@ public IllegalThreadException() {
     private static final int MINIMAL_DELAY = 5;
 
     /**
-     * Parameterless version of realsync which uses default timout (see DEFAUL_WAIT_TIME).
+     * Parameterless version of realsync which uses default timeout (see DEFAULT_WAIT_TIME).
```

Comment:
```suggestion
     * Parameterless version of realsync which uses the {@link #DEFAULT_WAIT_TIME default timeout}.
```

Reply:
I'm for a clearer message:

Parameterless version of `{@code realsSync}` which uses _the_ default timeout of `{@link #DEFAULT_WAIT_TIME}`.

Let us address this in a separate issue.

Reply:
[JDK-8294255](https://bugs.openjdk.org/browse/JDK-8294255): Add link to DEFAULT_WAIT_TIME in javadoc for SunToolKit.realsSync

Feel free to provide a patch.